tests/events/client/instrumentation/test_task_run_state_change_events.py

```diff
@@ -139,7 +139,7 @@ def happy_path():
             },
             "flow_run_run_count": 1,
             "name": task_run.name,
-            "run_count": 1,
+            "run_count": 0,
```

Comment:
wait isn't this test showing that we went from PENDING -> RUNNING -> COMPLETED? if so, why shouldn't this be 1?

Reply:
This is when it transitions from `PENDING` to `RUNNING` so the user code hasn't been run yet. When it transitions from `RUNNING` to `COMPLETED`, then the `run_count` is 1. I think we were incrementing the run count too early before.

Reply:
ahh - i see!